### ISSUE TITLE
gcc 4.9.1 warning

### ISSUE THREAD

**Issue by psychocoderHPC** [pngwriter/pngwriter#51]
I get the following warning when I compile with gcc `4.9.1` and use libpng version `1.6.17`

```
/home/slpp021/workspace/pngwriter/src/pngwriter.cc: In member function ‘void pngwriter::close()’:
/home/slpp021/workspace/pngwriter/src/pngwriter.cc:1081:11: warning: ‘const char* png_convert_to_rfc1123(png_structrp, png_const_timep)’ is deprecated (declared at /home/slpp021/lib/libpng-1.6.17/include/png.h:1175) [-Wdeprecated-declarations]
           png_convert_to_rfc1123(png_ptr, &mod_time),
           ^
/home/slpp021/workspace/pngwriter/src/pngwriter.cc:1081:52: warning: ‘const char* png_convert_to_rfc1123(png_structrp, png_const_timep)’ is deprecated (declared at /home/slpp021/lib/libpng-1.6.17/include/png.h:1175) [-Wdeprecated-declarations]
           png_convert_to_rfc1123(png_ptr, &mod_time),
```

**Comment by ax3l** [comment]
Thanks for the report!
Which branch did you use?

**Comment by ax3l** [comment]
no it's a new warning for the `libpng 1.6.X` branch: https://travis-ci.org/pngwriter/pngwriter/jobs/52386923

**Comment by ax3l** [comment]
yes, a duplicate of #11. I will close this since it's a know issue since this function is deprecated in 1.6 onwards :)

**Comment by ax3l** [comment]
Fixed in #54 :)

**Comment by psychocoderHPC** [comment]
:+1:

**Comment by ax3l** [comment]
nevertheless, when you compile `libpng` yourself, use the `1.2.X` series.

Especially the `1.6.X`+ support is still very fresh and I just found an other bug in pngwriter that will first be fixed with #54.

**Comment by psychocoderHPC** [comment]
Thanks for the information, I will switch to an older libpng version.
